widget/balance/src/main/java/com/ivy/widget/balance/WalletBalanceWidget.kt

```diff
@@ -55,17 +55,17 @@ class WalletBalanceWidget(
         provideContent {
             val prefs = currentState<Preferences>()
             val appLocked = prefs[booleanPreferencesKey("appLocked")] ?: false
-            val balance = prefs[stringPreferencesKey("balance")] ?: "0.00"
+            val balance = prefs[doublePreferencesKey("balance")] ?: 0.00
             val currency = prefs[stringPreferencesKey("currency")] ?: "USD"
-            val income = prefs[stringPreferencesKey("income")] ?: "0.00"
-            val expense = prefs[stringPreferencesKey("expense")] ?: "0.00"
+            val income = prefs[doublePreferencesKey("income")] ?: 0.00
```

Comment:
Use different names for the keys so there are no crashes like "income_v2", otherwise it'll crash for existing users